lib/widgets/gridcards.dart

```diff
@@ -0,0 +1,334 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:sochem/utils/constants.dart';
+
+class HomeScreenGrid extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Padding(
+              padding:
+                  const EdgeInsets.only(top: 20.0, left: 15.0, right: 15.0),
+              child: Container(
+                height: MediaQuery.of(context).size.height * 0.15,
+                width: MediaQuery.of(context).size.width * 0.40,
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(25),
+                  color: cardImageColor,
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.black45,
+                      offset: const Offset(
+                        1.0,
+                        1.0,
+                      ), //Offset
+                      blurRadius: 5.0,
+                    ),
+                  ],
+                ),
+                child: Column(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(top: 10.0),
+                      child: Center(
+                        child: Image.asset(
+                          'assets/feed.png',
+                          width: 60.0,
+                          height: 60.0,
```

Comment:
Same as above, use images path constants instead of directly using image paths here.

Reply:
Done